.github/workflows/sanity-test.yaml

```
@@ -108,12 +108,12 @@ jobs:
           --user "${CORSO_M365_TEST_USER_ID}" \
            --hide-progress 2>&1 | tee $TEST_RESULT/backup_exchange.txt 
 
-          if ! grep -q 'Completed (0 errors)' $TEST_RESULT/backup_exchange.txt
+          if ! grep -q 'Completed' $TEST_RESULT/backup_exchange.txt
```

I think we should check on the presence of errors here.  `Completed (99 errors)` is probably not a case we want to pass.
```suggestion
          if grep -q 'errors' $TEST_RESULT/backup_exchange.txt
```

---

I think a better way to go here is going to be to switch to `--json` output and parse the structured output. Going to file a ticket for that and we can address this follow up there.